### DISCUSSION TITLE
First Time panel builder, odd behaviour with CC output

### DISCUSSION THREAD

**Discussion by fatalgherkin** [RomanKubiak/ctrlr#571] [General]
Hello there, 
It's weird, as I started last night trying a few odd controls and these seemed to work ok.
So this morning I started on the graphics and control placement and spent quite a bit of time getting it looking right.
Built the EXE and.... 
Only the Ring Mod switch and the keyboard output work... Checked with the panel that worked last night and nothing seems to be different from the controller rotarys on the new..
If anyone can help it may be a bit obvious to some of you old hands, but I think its quite cool to be able to make your own controllers... 

Hopefully the first for this new Behringer JT4000 device ?
Attached the .panel file for you and also the graphics and ctrlr version..

Any help and tips for this noob would be really helpful.
Cheers and thanks in advance,
Andy
[jt4000-controller.zip](https://github.com/RomanKubiak/ctrlr/files/12906658/jt4000-controller.zip)
![CTRLR Version](https://github.com/RomanKubiak/ctrlr/assets/63640942/118667fd-c1fc-48e0-9b3f-e08f1acfb544)
![panel graphics](https://github.com/RomanKubiak/ctrlr/assets/63640942/84b7ac8e-ff9b-465b-a3ea-9a0943817f14)

**Comment by fatalgherkin** [comment]
Using the output midi monitor... The other controls were moved inbetween the two..
![working bits](https://github.com/RomanKubiak/ctrlr/assets/63640942/c4b06eb6-99d1-4564-a035-eeaf968a7200)

**Comment by fatalgherkin** [comment]
Controller values as per this sheet..
![16973166131342462603686641169763](https://github.com/RomanKubiak/ctrlr/assets/63640942/9ae015f4-738e-4f55-a0f2-a1bcafdc9a87)

**Comment by dnaldoog** [comment]
Hi there @fatalgherkin  (Andy)!

First thing is to ditch that version of Ctrlr.

you are best to use version 5.3.201 otherwise it will be a case of fatal gherkinism.

**Reply by fatalgherkin** [reply]
No problemo, will download and try the prior version. Hopefully that will fix the issues...Many thanks for your advice.

**Reply by fatalgherkin** [reply]
From https://ctrlr.org/nightly/... Gotcha.

**Comment by dnaldoog** [comment]
Great!

Also look at [this](https://github.com/RomanKubiak/ctrlr/discussions/509#discussioncomment-5099990) panel for various ideas including solving the panel being truncated by 24 px when exporting as exe. That code should be in  init() function which runs when the panel loads.
